handlesErrors.js

```diff
@@ -0,0 +1,25 @@
+const handles404Errors = (err, req, res, next)=>{
+    if(err.status && err.msg){
```

Comment:
Lets have a look at how we can check for an invalid path and move this into app.js :)